File: 1-two-sum/two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
      Dictionary<int,int>  mp = new  Dictionary<int,int>();

      for(int i= 0 ; i < nums.Length ; i++){
          if(mp.ContainsKey(target - nums[i])){
              return new int[]{mp[target-nums[i]]  , i };
          }
          mp.TryAdd(nums[i],i);
      }

      return default;
    }
}

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> mp= new HashMap<>();
        int length = nums.length;
        for(int i = 0 ; i < length ; i++){
            if(mp.containsKey(target - nums[i])){
                return new int[]{i, mp.get(target-nums[i])};
            }
            mp.put(nums[i], i);
        }
        return new int[]{0,0};
    }
}